aria/cli/core/aria.py

```diff
@@ -65,15 +58,33 @@ def __init__(self, *args, **kwargs):
         super(MutuallyExclusiveOption, self).__init__(*args, **kwargs)
 
     def handle_parse_result(self, ctx, opts, args):
-        if self.mutually_exclusive.intersection(opts) and self.name in opts:
+        if (self.name in opts) and self.mutually_exclusive.keys().intersection(opts):
```

Comment:
seems like `mutually_exclusive` might be a `tuple` now, so `.keys()` would raise an error here.